### ISSUE TITLE
Try to train the model on DNR dataset

### ISSUE THREAD

**Issue by LuckyHeart** [starrytong/SCNet#26]
Excellent work!!

I am trying to train the model on the DNR dataset. I am using the default configuration and training the model on a single GPU.

I set the learning rate to 0.0001.
I trained for 5 epochs and tested the model, but I found that the model's output contains significant noise. Could you please help me understand whether this is due to the limited number of training epochs or if it is related to the training parameter settings?

**Comment by starrytong** [comment]
Training for 5 epochs with a lower learning rate might not be sufficient. Perhaps you could try increasing the learning rate and continue training.
